tests/core/eth-module/test_accounts.py

```diff
@@ -371,3 +379,52 @@ def test_eth_account_prepared_encrypt(acct, web3js_key, web3js_password):
     decrypted_key = acct.decrypt(encrypted, web3js_password)
 
     assert decrypted_key == to_bytes(hexstr=web3js_key)
+
+
+@pytest.mark.xfail
+@pytest.mark.parametrize(
+    'txn, private_key, expected_raw_tx, tx_hash, r, s, v',
```

Comment:
A couple of these variable names could be updated in this context:

- `txn` -> `expected_txn`
- `expected_raw_tx` -> `raw_txn`
- `private_key` can be deleted
- `tx_hash` -> `expected_txn_hash`